cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -346,11 +346,7 @@ class DownloaderCommand : OrtCommand(
         } catch (e: DownloadException) {
             e.showStackTrace()
 
-            val failureMessage = "Could not download '${pkg.id.toCoordinates()}': " +
-                    e.collectMessages()
-            failureMessages += failureMessage
-
-            logger.error { failureMessage }
+            failureMessages += e.collectMessages()
```

Comment:
Commit message: "as the happen" -> "when they are happening"?

Reply:
Corrected to say "as they happen".